### DISCUSSION TITLE
No component was found at offset 71

### DISCUSSION THREAD

**Discussion by charlesanim** [fwouts/previewjs#925] [General]
When I Click on vscode extenstion, i get `No component was found at offset 71`. Any idea whats going on here. 

I'm seeing this error in pure react components as well as storybook stories

**Comment by fwouts** [comment]
@charlesanim to be clear, are you clicking on the "P" button when your cursor is positioned within the definition of a React component?

**Reply by charlesanim** [reply]
yes. Should I click and stay on the P button? Have you seen this issue before?

**Reply by fwouts** [reply]
No, that's a new one. Would you be able to file an issue including a minimal repro?

**Reply by charlesanim** [reply]
sure, i'll file an issue soon. thank you

**Comment by LaustinSpayce** [comment]
I am also having this issue, let me see if I can make a minimal repro

**Reply by LaustinSpayce** [reply]
I initialised a create-react-app with typescript and redux by using the following command:
`npx create-react-app my-app --template redux-typescript`
And right out the box in this barebones project I get "no component found at offset xyz".

**Comment by Tonisenpai** [comment]
Mine is saying "no component was found at offset 0". It won't run, neither is the "P" button showing up anywhere in the vs code user interface.

**Reply by fwouts** [reply]
Hey @Tonisenpai, would you be able to try the CLI to see what might be going wrong?

You can run it with debug logs as follows:
```
PREVIEWJS_LOG_LEVEL=debug npx @previewjs/cli
```

Feel free to open an issue and share a sample repo too.

**Comment by leb24551347** [comment]
> When I Click on vscode extenstion, i get `No component was found at offset 71`. Any idea whats going on here.
> 
> I'm seeing this error in pure react components as well as storybook stories

I have the same issue when I run the  "open preview.js", but after I reload the vscode(ctrl+ shift + P -> reload window) I see the preview function in my  component
![螢幕擷取畫面 2023-06-23 093821](https://github.com/fwouts/previewjs/assets/80091009/4a1e3729-39cc-44d5-a756-05c5d9000a1b)
just click the "open xxxx in Preview js", you will see the preview window.
